README.md

```diff
@@ -1,73 +1,32 @@
+Scala using [Scalameta](http://scalameta.org/)-related technologies.
 
-:warning: This project is very alpha stage.
-Expect bugs and surprising behavior.
-Ticket reports and patches are welcome!
+:warning: This project is under development and is not intended to be used for
+day-to-day coding. Expect bugs and incomplete documentation. Installation
+instructions are primarily intended for project contributors.
 
 ## Project Goals
 
-This project has the following goals:
-
-- a good UX for the final users, including simple installation and setup
-- low memory requirements
-- integration with scalameta-based tools, such as [Scalafix](https://github.com/scalacenter/scalafix) and [Scalafmt](https://github.com/scalameta/scalafmt)
-
-## Roadmap
-
-Below is a rough summary of what features have been implemented.
-Even if some checkbox is marked it does not mean that feature works perfectly.
-Some of those features are a likely outside the scope of this project, we are
-still learning and exploring what's possible.
-
-* Compile errors with the Scala Presentation Compiler (`textDocument/publishDiagnostics`):
-  - [ ] On build compile
-  - [x] As you type
-* Integration with the sbt server:
-  - [x] Run custom sbt command on save
-  - [x] Receive compilation errors (`textDocument/publishDiagnostics`)
-* Linting with Scalafix (`textDocument/publishDiagnostics`):
-  - [x] On build compile
-  - [x] As you type
-* Refactoring with Scalafix:
-  - [ ] Quick-fix inspections (`textDocument/codeAction`)
-    + [x] Remove unused imports
-  - [x] Rename local symbol (`textDocument/rename`)
-  - [ ] Rename global symbol (`textDocument/rename`)
-* Formatting with Scalafmt:
-  - [x] Whole file (`textDocument/formatting`)
-  - [ ] Selected range (`textDocument/rangeFormatting`)
-  - [x] On save (`textDocument/willSaveWaitUntil`)
-  - [ ] As you type (`textDocument/onTypeFormatting`)
-* Code assistance:
-  - [x] Auto-complete symbols in scope as you type (`textDocument/completions`)
-  - [ ] Auto-complete global symbol and insert missing imports (`textDocument/completions`)
-  - [x] Show parameter list as you type (`textDocument/signatureHelp`)
-  - [x] Show type at position (`textDocument/hover`)
-* Go to definition with SemanticDB (`textDocument/definition`):
-  - [x] Inside the project
-  - [x] From project files to Scala dependency source files
-  - [x] From project files to Java dependency source files
-  - [ ] From project dependency to project dependency
-* Find references with SemanticDB (`textDocument/references`):
-  - [x] In file (`textDocument/documentHighlight`)
-  - [x] In project
-  - [ ] In dependencies
-* Lookup symbol definition by name:
-  - [x] In file (`textDocument/documentSymbol`)
-  - [x] In workspace (`workspace/symbol`)
-* Symbol outline:
-  - [x] In file as you type (`textDocument/documentSymbol`)
+* simple installation
+* correct diagnostics (red squiggles)
+* robust navigation (goto definition anywhere)
+* fast completions
```

Comment:
"fast _context-aware_ completions"?

Reply:
Do people distinguish between the two? I think "completions" is clear enough.

Reply:
Well, all this time working with Scala, I was using fast completions, but it wasn't context-aware, just anything from the open buffers. I don't care much about clarifying this, just suggested it because it would make a difference for me reading this for the first time. Probably you're right that in this context it's obvious enough.